jenkins/gce-custom-images/create_gce_image.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+# This script is a wrapper around the command that runs the custom GCE image
+# creation tool - Daisy - for local development and documentation purposes.
+# It should be run from the root of the Leonardo repo.
+# It assumes that a Daisy image was pulled; e.g. docker pull gcr.io/compute-image-tools/daisy:latest
+
+# The date-time suffix is in the format yymmdd-hh-mm but it could be anything.
+OUTPUT_IMAGE_NAME="leo-custom-gce-image-200219-16-05"
```

Comment:
Eventually this will be Jenkins-ized and these params will turn into Jenkins input parameters, right?

Reply:
right